README.md

```diff
@@ -63,3 +63,92 @@ Most of the translations can be found in the web client but we have several othe
 <a href="https://translate.jellyfin.org/engage/jellyfin/?utm_source=widget">
 <img src="https://translate.jellyfin.org/widgets/jellyfin/-/jellyfin-web/multi-auto.svg" alt="Detailed Translation Status"/>
 </a>
+
+## Development
+
+These instructions will help you get set up with a local development environment in order to contribute to this repository. Before you start, please be sure to completely read our [guidelines on development contributions](https://jellyfin.org/docs/general/contributing/development.html). Note that this project is supported on all major operating systems (Windows, Mac and Linux).
+
+### Prerequisites
+
+The following software prerequisites are required to be installed locally before the project can be built and executed.
+
+* [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download)
+* [Visual Studio](https://visualstudio.microsoft.com/downloads/) (at least version 2017) or [Visual Studio Code](https://code.visualstudio.com/Download)
```

Comment:
Is this for windows or when you want to run the web client in a dev environment? as it reads, it makes it sound like these are hard coded requirements when they're not

Reply:
Updated this section to describe IDEs as optional prerequisites only required for debugging